tests/dummy/app/templates/disabled.hbs

```diff
@@ -1,4 +1,4 @@
 <div id="custom-disabled">
   {{input value=model.name}}
-  {{async-button promiseBinding=promise disableWhen=model.isNotDirty default="Save" pending="Saving..."}}
```

Comment:
Good find.  I didn't realize we were still using `fooBinding` syntax in the template here.  This syntax doesn't work properly in 2.0 (and we were just using a bound param anyways so it isn't even needed).